### PR TITLE
Add basic Documenter workflow

### PR DESCRIPTION
**Pull request by sloede**
Should be merged after #2.

After #2 has been merged, the `README.md` in this PR here can be discarded (just a placeholder). Also, the file `LICENSE` should be renamed to `LICENSE.md`

### CODE REVIEW

**Comment by sloede** [comment]
Ready for review...

File: README.md [hunk 1]
```diff
@@ -72,11 +72,12 @@ through Conduit nodes, which are required to adhere to the
...
 A user-defined ParaView pipeline script can be passed to `catalyst_initialize` using the
-keyword argument `catalyst_pipeline`. By default src/catalyst_pipeline.py is used. This
-script activates live visualization. The data can also be viewed on a remote machine. In
-this case `options.CatalystLiveURL` has to be adjusted to point to the remote machine.
+keyword argument `catalyst_pipeline`. By default src/catalyst_pipeline.py is used, which
+activates live visualization. The data can also be viewed on a remote machine. In this case
+`options.CatalystLiveURL` has to be adjusted to point to the remote machine.
 Once a pipeline in ParaView has been set up, it can be exported as a script by selecting
-"File" from the main menu and then "Save Catalyst State".
+"File" from the main menu and then "Save Catalyst State". It might be required to modifiy
```
Comment (sloede) [inline README.md]:
```suggestion
"File" from the main menu and then "Save Catalyst State". It might be required to modify
```

**Review by sloede — COMMENTED** [review]
LGTM now!

**Comment by benegee** [comment]
Thank a lot @sloede !
(and thanks for fixing the Catalyst\_jll link!)

**Review by benegee — APPROVED** [review]

**Pull request merged**